react/categoryMenu.css

```diff
@@ -34,4 +34,12 @@
 
 .menuContainer {}
 
-.sidebarItemContainer {}
\ No newline at end of file
+.sidebarItemContainer {}
+
+.sidebarItem:not([expanded=""]) > div > div > span > .iconOpen {
```

Comment:
Why not just 

```css
.sidebarItem:not([expanded=""]) .iconOpen
```

Reply:
because I want to affect only the icon for that specific `.sidebarItem`, and not all icon inside it. one issue you might see with this approach is to imagine multiple nested accordions.

Reply:
I think we should add a CSS handle for that. That CSS HTML element nesting is what we are trying to forbid in styles builder because it's fragile.

Reply:
I already have a CSS class for that, it's the `.iconOpen` and `.iconClose`, but it doesn't work if I just use `.sidebarItem[expanded=""] .iconClose` (and vice-versa), because it will affect **all** icons inside the `.sidebarItem`, and I want to affect only the icon "related" to that sidebar item.

will it only be forbidden in the `styles` builder? I want to use this directly in the module of the own component, so it should be fine, right?